tomb-wasm/Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde = { workspace = true }
 tomb-crypt = { workspace = true }
 wnfs = { workspace = true }
 blake3 = { workspace = true }
+uuid = { workspace = true }
```

Comment:
Only enforcing this on new dependencies, we'll want to clean up the old ones eventually but yeah as I said elsewhere, for new dependencies please don't add them workspace wide.